src/Microsoft.Content.Build.MerfToSDPForJava.DataContracts/ViewModel/ArticleItemYaml.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.DataContracts
```

Comment:
suggest we put Merf data model and SDP data model to different folder which will be easier identify

Reply:
sure

Reply:
Updated